content/actions/creating-actions/creating-a-composite-run-steps-action.md

```diff
@@ -40,8 +40,14 @@ Before you begin, you'll create a {% data variables.product.product_name %} repo
 3. From your terminal, make `goodbye.sh` executable.
 
   ```shell
+  # bash
   chmod +x goodbye.sh
   ```
+  
+  ```powershell
+  # Windows PowerShell
+  git update-index --chmod=+x ./goodbye.sh
```

Comment:
Would this command also work identically under macOS/Linux? If so, it would be great if we could just replace the original `chmod` command for this more widely compatible alternative.